tests/src/test/java/alluxio/master/MasterFaultToleranceIntegrationTest.java

```diff
@@ -229,20 +242,29 @@ public void killStandby() throws Exception {
 
   @Test
   public void workerReRegister() throws Exception {
+    Assume.assumeFalse(HadoopClientTestUtils.isHadoop1x());
     AlluxioBlockStore store = AlluxioBlockStore.create();
     Assert.assertEquals(WORKER_CAPACITY_BYTES, store.getCapacityBytes());
 
     for (int kills = 0; kills < MASTERS - 1; kills++) {
       Assert.assertTrue(mMultiMasterLocalAlluxioCluster.stopLeader());
       mMultiMasterLocalAlluxioCluster.waitForNewMaster(CLUSTER_WAIT_TIMEOUT_MS);
-      waitForWorkerRegistration(store, 1, 5 * Constants.SECOND_MS);
-      // If worker is successfully re-registered, the capacity bytes should not change.
-      Assert.assertEquals(WORKER_CAPACITY_BYTES, store.getCapacityBytes());
+      long capacityFound = -1;
+      for (int i = 0; i < MASTERS; ++i) {
```

Comment:
This was used in -Phadoop1 where a master can be transiently elected as master.  This was what happened:

1. Master A (leader) is stopped.
2. Master A is elected as leader.
3. Master A is shutdown because of the loop in LocalAlluxioMaster#stop. 

This only happened in hadoop1. I have not figured out why step 2 happened. Since we have disabled this test in hadoop1 profile, i am fine with removing it if you prefer.